recipe/meta.yaml

```diff
@@ -12,30 +12,36 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
     - python
     - setuptools
-    - numpy x.x
+    - numpy 1.8.*  # [not (win and (py35 or py36))]
+    - numpy 1.9.*  # [win and py35]
+    - numpy 1.11.*  # [win and py36]
```

Comment:
@kain88-de I guess that this is the current combination. I don't really want to spread this at the moment. I'd rather work fast to get older versions of `numpy` in the channel so we can simplify this ASAP.